src/opera/commands/deploy.py

```diff
@@ -20,32 +20,48 @@ def add_parser(subparsers):
     )
     parser.add_argument(
         "--inputs", "-i", type=argparse.FileType("r"),
-        help="YAML file with inputs",
+        help="Optional: YAML file with inputs to override the inputs supplied in init",
     )
-    parser.add_argument("csar",
-                        type=argparse.FileType("r"),
-                        help="cloud service archive file")
+    parser.add_argument("template",
+                        type=argparse.FileType("r"), nargs='?',
+                        help="TOSCA YAML service template file",
+                        )
     parser.set_defaults(func=deploy)
 
 
 def deploy(args):
     if args.instance_path and not path.isdir(args.instance_path):
         raise argparse.ArgumentTypeError("Directory {0} is not a valid path!".format(args.instance_path))
 
-    storage = Storage(Path(args.instance_path).joinpath(".opera")) if args.instance_path else Storage(Path(".opera"))
-    storage.write(args.csar.name, "root_file")
+    storage = Storage(Path(args.instance_path) / ".opera") if args.instance_path else Storage(Path(".opera"))
```

Comment:
```suggestion
    storage = Storage(Path(args.instance_path)) if args.instance_path else Storage(Path(".opera"))
```
Suggestion as discussed in the main thread.

Reply:
Done.